esp/bindings/SOAP/Platform/soapbind.hpp

```diff
@@ -148,6 +151,12 @@ class esp_http_decl CSoapRequestBinding : public CSoapComplexType,
     void setThunkHandle(void * val){thunk_=val;}
     void * getThunkHandle(){return thunk_;}
 
+    void setConnectTimeOutMs(unsigned timeout){connectTimeoutMs = timeout;}
```

Comment:
Ah, well that answers my first question "or maybe someone already has?".  Perfect.